zellij-utils/assets/config/log4rs.yml

```diff
@@ -0,0 +1,46 @@
+# Scan this file for changes every 30 seconds
+refresh_rate: 30 seconds
+
+appenders:
+  # An appender named "stderr" that writes to stderr
+  stderr:
+    kind: console
+    target: stderr
+
+  # default zellij appender, should be used across most of the codebase.
+  logFile:
+    kind: file
+    path: "zellij.log"
+    append: false
+    encoder:
+      # {n} means platform dependent newline
+      # module is padded to *at least* 25 bytes and thread is padded to be between 10 and 15 bytes.
+      pattern: "|{module:<25}| {date} {highlight({level})} [{thread:<10.15}] [{file}:{line}]: {message} {n}"
+
+  # plugin appender. To be used in decorating_pipe to forward stderr output from plugins.
+  logPlugin:
+    kind: file
+    path: "zellij.log"
+    encoder:
+      # {n} means platform dependent newline
+      pattern: "{message} {n}"
```

Comment:
I think we should make this as similar as possible to the `logFile` pattern, then use [`log!`](https://docs.rs/log/0.4.14/log/macro.log.html) in the `DecoratedPipe`.

We should at least include the `date` and maybe even `level` if possible.

Additionally, are there different (shorter) ways to format the `date`?

Reply:
We definitely can format date differently! What is the smallest unit we want to include? Milliseconds?

Reply:
I went with hybrid approach. I moved `level` to the beginning of log message but still do some formatting in `logging_pipe`. This way logs from plugin have `plugin_name` instead of `module` and `plugin_id` instead of `thread`.